src/views/Login.js

```diff
@@ -61,37 +59,54 @@ export default function Login() {
   return (
     <View pop={pop} inset>
       <Grid>
-        <Grid.Item full as={Crumbs} routes={[{ text: 'Bridge' }]} />
-        <Grid.Item full as={H4} className="mt4">
-          Login
-        </Grid.Item>
-
-        <Grid.Item
-          full
-          as={Tabs}
-          className="mt1"
-          // Tabs
-          views={VIEWS}
-          options={OPTIONS}
-          currentTab={currentTab}
-          onTabChange={setCurrentTab}
-          // Tab extra
-          goHome={goHome}
-        />
-        <Grid.Item full as={P} className="f6">
-          Version {version}
+        <Grid.Item full as={Text} className="flex justify-center mt9 mb7">
+          <Grid.Item as={Text} className="gray3">
+            Urbit ID /&nbsp;
+          </Grid.Item>
+          <Grid.Item as={Text}>Login</Grid.Item>
         </Grid.Item>
+        {isOther && <Grid.Item full as={Other} goHome={goHome} />}
+        {!isOther && <Grid.Item full as={Ticket} goHome={goHome} />}
+        {!isOther && (
+          <>
+            <Grid.Item full className="t-center mv4 gray4">
+              or
+            </Grid.Item>
+            <Grid.Item
+              full
+              as={Button}
+              className="b-solid b1 b-black"
+              center
+              onClick={() => setisOther(true)}>
+              Metamask, Mnemonic, Hardware Wallet...
+            </Grid.Item>
+            <Grid.Item full onClick={goToActivate} className="mv10 t-center f6">
+              <span className="gray4">New Urbit ID? </span>
+              <LinkButton>Activate</LinkButton>
+            </Grid.Item>
+          </>
+        )}
+        {isOther && (
+          <Grid.Item
+            as={LinkButton}
+            onClick={() => setisOther(false)}
+            full
+            className="t-center underline f6 mt8">
+            Back
```

Comment:
Should this just say "Master Ticket" or similar instead, to have more symmetry with the "Metamask, Mnemonic..." button?